web/js/projection/projection.model.js

```diff
@@ -1,20 +1,16 @@
-/**
- * @module wv.proj
- */
-var wv = wv || {};
-wv.proj = wv.proj || {};
+import each from 'lodash/each';
+import util from '../util/util';
+import ol from 'openlayers';
+import proj4 from 'proj4';
 
-/**
- * @class wv.proj.model
- */
-wv.proj.model = wv.proj.model || function (config) {
+export function projectionModel(config) {
```

Comment:
What do you think about keeping the name `model`, importing this into the `proj` parent, and attaching it to the object that exports?